advocacy_docs/community/contributing/styleguide.mdx

```diff
@@ -656,4 +657,20 @@ Examples: EDB Postgres® AI, EDB Postgres® Analytics, EDB Postgres® Machine Le
 
 For other trademarks, the documentation site already includes a general disclaimer at the footer of every page called "Trademark" which links to https://www.enterprisedb.com/trademarks.
 Contrary to the specifications on [Trademarks](https://www.enterprisedb.com/trademarks), don't include the symbol in page titles, 
-because the symbol can affect how search engines index and find pages.
\ No newline at end of file
+because the symbol can affect how search engines index and find pages.
+
+## Frontmatter 
+
+The frontmatter is located at the top of every markdown page, and allows you to specify metadata and options in YAML language. Frontmatter lets you control how a page will render.
+
+Ensure your page, at minimum, includes a `title` and `description` as metadata. It can include other metadata such as `navTitle`, `deepToC`, etc. 
+
+```yaml
+---
+title: Working with the Repo
+description: Step by step instructions to install and use the docs repo.
+---
+```
+
+!!!note
+   Ensure the `description` has a final stop, even if it isn't a full grammatical sentence. Descriptions are used to dynamically build a summary for each section in `index` pages. Since all descriptions are then displayed next to each other, add a full stop to ensure visual consistency.
```

Comment:
```suggestion
   Ensure the `description` has a final stop (period), even if it isn't a full grammatical sentence. Descriptions are used to dynamically build a summary for each section in `index` pages. Since all descriptions are then displayed next to each other, add a full stop to ensure visual consistency.
```